api/tests/routes/pro/test_offerers_addresses.py

```diff
@@ -0,0 +1,260 @@
+        foreign_offerer = offerers_factories.OffererFactory()
+        foreign_offerer_id = foreign_offerer.id
+
+        expected_data = {
+            "label": "Label",
+            "banId": "75108_5194_00089",
+            "inseeCode": "75108",
+            "street": "89 Rue de la Boétie",
+            "postalCode": "75008",
+            "city": "Paris",
+            "latitude": 48.87171,
+            "longitude": 2.30829,
+        }
+
+        http_client = client.with_session_auth(pro_user.email)
+
+        with assert_num_queries(4):
+            response = http_client.post(
+                f"/offerers/{foreign_offerer_id}/addresses/",
+                json={
+                    "label": expected_data["label"],
+                    "street": expected_data["street"],
+                    "inseeCode": expected_data["inseeCode"],
+                },
+            )
+
+        assert response.status_code == 403
+        assert not geography_models.Address.query.count()
+        assert not offerers_models.OffererAddress.query.count()
+
+    @override_settings(ADRESSE_BACKEND="pcapi.connectors.api_adresse.ApiAdresseBackend")
+    @pytest.mark.usefixtures("db_session")
+    def test_check_consistency_betwenn_address_and_coordinates(self, client, requests_mock):
+        payload = {
+            "label": "Label",
+            "banId": "75118_2974_00018",
+            "inseeCode": "75118",
+            "street": "18 Rue Duhesme",
+            "postalCode": "75018",
+            "city": "Paris",
+            "latitude": 48.89078,
+            "longitude": -15.30829,  # Dans l'océan atlantique
+        }
+
+        expected_data = {
+            "label": "Label",
+            "banId": "75118_2974_00018",
+            "inseeCode": "75118",
+            "street": "18 Rue Duhesme",
+            "postalCode": "75018",
+            "city": "Paris",
+            "latitude": 48.89079,
+            "longitude": 2.33856,
+        }
+        requests_mock.get(
+            f"""https://api-adresse.data.gouv.fr/search?q={payload["street"]}&citycode={payload["inseeCode"]}&autocomplete=0&limit=1""",
+            json=fixtures.ONE_FEATURE_RESPONSE,
+        )
+        pro_user = users_factories.ProFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerer_id = offerer.id
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
+
+        http_client = client.with_session_auth(pro_user.email)
+
+        # Fetch the session
+        # Fetch the user
+        # Check permissions
+        # Insert address
+        # Insert OffererAddress
+        # Select OffererAddress & address
+        with assert_num_queries(6):
+            response = http_client.post(
+                f"/offerers/{offerer_id}/addresses/",
+                json={
+                    "label": expected_data["label"],
+                    "street": expected_data["street"],
+                    "inseeCode": expected_data["inseeCode"],
```

Comment:
Si je comprends bien le test, ce serait plutôt : 
```suggestion
                    "label": payload["label"],
                    "street": payload["street"],
                    "inseeCode": payload["inseeCode"],
```
Mais finalement, les champs utilisés dans `payload` sont identiques à ceux de `expected_data`, car les coordonnées GPS ne font pas (ou plus) partie de `OffererAddressRequestModel`. Ainsi je ne saisis pas bien ce que montre le test.

Par contre, on pourrait envisager un test avec une rue qui n'existe pas dans la commune par exemple, ou un label incohérent avec la rue, du moins quelque chose qui renvoie un comportement autre que nominal de l'API Adresse (avec un json d'une vraie réponse).